src/ccache/argprocessing.cpp

```diff
@@ -756,6 +762,15 @@ process_option_arg(const Context& ctx,
     return Statistic::none;
   }
 
+  if (util::starts_with(arg, "-fcoverage-compilation-dir")) {
+    const auto eq_pos = arg.find('=');
+    if (eq_pos != std::string_view::npos) {
+      args_info.coverage_compilation_dir = arg.substr(eq_pos + 1);
+    }
```

Comment:
Could this be

```suggestion
  if (util::starts_with(arg, "-fcoverage-compilation-dir=")) {
    const auto eq_pos = arg.find('=');
    args_info.coverage_compilation_dir = arg.substr(eq_pos + 1);
```

?

Reply:
Sure.

Reply:
OK. Changed in [31a1e88](https://github.com/ccache/ccache/pull/1644/commits/31a1e8834fa7f50f5a32dfbaedcc6770a359b3a6).